dotcom-rendering/src/web/components/Dateline.tsx

```diff
@@ -50,12 +50,13 @@ const labelStyles = css`
 export const Dateline: React.FC<{
 	primaryDateline: string;
 	secondaryDateline: string;
-}> = ({ primaryDateline, secondaryDateline }) => {
+	cssOverrides?: SerializedStyles;
```

Comment:
We can replace this with a prop. it's true that we use `cssOverrides` in Source but the reason they exist there is because it's a shared design system and it's not possible to cover all use cases just using props alone. So they offer cssOverrides as an escape hatch.

But in DCR we have total control of the consumers of these components so there's no need to pass classnames as props. Instead, we can use props to create a readable, typeable api. The problem with having a classname as a prop is it is essentially allowing _any_ style to be set, it opens a floodgate, basically.

Reply:
Yeaa I had thought about this also. Thanks for the thorough explanation :)  I removed the `cssOverrides` and used `format` and `palette` as parameters for `Dateline` component.